workflow/scripts/plot_frip_score.R

```diff
@@ -0,0 +1,21 @@
+log <- file(snakemake@log[[1]], open="wt")
+sink(log)
+sink(log, type="message")
+
+library("tidyverse")
+
+data <- lapply(snakemake@input, read.table, header=F, stringsAsFactors = F)
+frip_scores <- tibble()
+for (i in 1:length(data)) {
+  frip_scores <- rbind(frip_scores, data[[i]])
+}
+names(frip_scores) <- c("sample_control", "frip")
```

Comment:
This is quite elegant, I should keep this pattern in mind for future use (aka as "steal it" :D).